build/packager.php

```diff
@@ -32,8 +32,8 @@
 $autoloaderContents = [
     'Aws/functions.php',
     'GuzzleHttp/functions_include.php',
-    'GuzzleHttp/Psr7/functions_include.php',
-    'GuzzleHttp/Promise/functions_include.php',
+    'GuzzleHttp/Psr7/Utils.php',
```

Comment:
It is not necessary to include these two utils files.

Reply:
@GrahamCampbell,

Happy to revert this if you wanted to give the PR another once-over.

Reply:
It's not a bug, just unnecessary to include these classes explicitly, since they already get autoloaded like the others. The original code was only necessary to include functions.

Reply:
We'll keep that in mind and will probably add something to our backlog for this.  If you notice anything else, definitely let us know.